README.md

```diff
@@ -77,35 +77,33 @@ client.connect().then(() => {
 #### Server
 
 ```js
-var Hapi = require('hapi');
-var Nes = require('nes');
+const Hapi = require('hapi');
+const Nes = require('nes');
 
-var server = new Hapi.Server();
+const server = new Hapi.Server();
 
-server.register(Nes).then(() => {
+server.register(Nes).then(async () => {
```

Comment:
I rather switch this to use await. Basically get rid of all the `then()` in the examples.

Reply:
OK, should I do this for the client examples as well?